src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -201,6 +201,7 @@ public boolean hasPerson(Customer customer) {
     @Override
     public void deletePerson(Customer target) {
```

Comment:
Can we rename it to `deleteCustomer`?

Reply:
Let's work on this in v1.4. There are traces of Person throughout the codebase that hasn't been updated.